cmd/kube-proxy/app/server.go

```diff
@@ -115,7 +117,8 @@ func NewProxyConfig() *ProxyServerConfig {
 		HealthzBindAddress: net.ParseIP("127.0.0.1"),
 		OOMScoreAdj:        qos.KubeProxyOOMScoreAdj,
 		ResourceContainer:  "/kube-proxy",
-		SyncPeriod:         30 * time.Second,
+		IptablesSyncPeriod: 30 * time.Second,
+		ConfigResyncPeriod: 10 * time.Minute,
```

Comment:
Would it be possible to randomize this number to avoid all KubeProxies resync at the same time?
Can we e.g. change the flag to e.g. MinConfigSyncPeriod and make this value to be random from: [MinConfigSyncPeriod, 2*MinConfigSyncPeriod]
?